src/controllers/queryRunner.ts

```diff
@@ -518,6 +516,23 @@ export default class QueryRunner {
         }
     }
 
+    /**
+     * Gets the first column where the selection started from
+     */
+    private getFirstSelectionColumn(rowIdToRowMap: Map<number, DbCellValue[]>, rowIdToSelectionMap: Map<number, ISlickRange[]>): number {
```

Comment:
You should do the same for the trailing tabs as well.

Reply:
we don't hae trailing spaces after the last column, so this is not an issue there

Reply:
I'm saying adding tabs for rows that don't have the "last" column selected. 

Repro :

![image](https://user-images.githubusercontent.com/28519865/72370640-8c5c5d80-36b7-11ea-9e80-1716766aced0.png)

I expect the copied text to be

```
name<tab>database_id
master<tab>
<tab>2
```

But with your change it will only this

```
name<tab>database_id
master
<tab>2
```

This isn't as big of a deal as the leading tabs, but we should be consistent with how we treat "empty" columns.

Reply:
Added tabs in the end as well